src/main/resources/application.yml

```diff
@@ -1,5 +1,18 @@
 
 spring:
+  data:
+    redis:
+      repositories:
+        enabled: false
+  redis:
+    host: ${REDIS_HOST:localhost}
```

Comment:
I take we need to define these new env variables in Terraform scripts? @michalpiatek

Reply:
yip - after verification I guess